.travis.yml

```diff
@@ -1,25 +1,45 @@
+dist: xenial
```

Comment:
`xenial` is old, let's use at least `bionic` (18.04)

Reply:
@januszm right. Now that the EOL ruby versions have been removed we can also go for `Focal`